grpc.go

```diff
@@ -0,0 +1,47 @@
+package core
+
+import (
+	"context"
+	"fmt"
+	"net"
+
+	"github.com/sirupsen/logrus"
+	"go.uber.org/fx"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/reflection"
+)
+
+var GrpcModule = fx.Provide(
+	newGrpc,
+)
+
+func newGrpc(lifecycle fx.Lifecycle, logger *logrus.Logger) *grpc.Server {
+	addr := ":9000"
```

Comment:
Do we want to configure the gRPC port by an env variable? We could still use `9000` as a default